### PR TITLE
Changes to websites.md

### PR DESCRIPTION
**Pull request by emceeaich**
I was using these directions to publish a static site at a domain, and wanted to update them:

* Break up into sections 
* Add more documentation on how to set up DNS records
* Clean up language

I followed the documentation in https://docs.ipfs.io/guides/concepts/dnslink/ while writing this.

### CODE REVIEW

**Comment by schomatis** [comment]
@emceeaich Sorry for the delay, could you rebase to resolve the conflict?

**Comment by emceeaich** [comment]
Hi, I'll work on this right away. I just got back from three weeks of travel. Thanks!

File: content/guides/examples/websites.md [hunk 1]
```diff
@@ -20,63 +26,104 @@ added QmYh6HbZhHABQXrkQZ4aRRSoSa6bb9vaKoHeumWex6HRsT mysite/index.html
...
 
-The very last hash next to the folder name is the one you want, let's call it
+The last hash, next to the folder name, `mysite/` is the one to remember, call it
 `$SITE_HASH` for now.
 
-Now, you can test it out locally by opening `http://localhost:8080/ipfs/$SITE_HASH`
-in your web browser! Next, to view it coming from another ipfs node, you can try
-`http://gateway.ipfs.io/ipfs/$SITE_HASH`. Cool, right?  But those hashes are
-rather ugly. Let's look at some ways to get rid of them.
+You can test it out locally by opening `http://localhost:8080/ipfs/$SITE_HASH`
```
[the remark this answers is not in the document]
Comment (emceeaich) [reply content/guides/examples/websites.md]: Made that change.

**Comment by olizilla** [comment]
Thanks for this @emceeaich this is great!

LETS GET IT MERGE IT!

**Pull request merged**